screenpipe-app-tauri/src-tauri/src/main.rs

```diff
@@ -337,6 +384,86 @@ async fn main() {
             #[cfg(target_os = "macos")]
             app.set_activation_policy(tauri::ActivationPolicy::Regular);
 
+
+            #[cfg(desktop)]
+            {
+                let toggle_recording_shortcut = Shortcut::new(Some(Modifiers::CONTROL | Modifiers::SHIFT), Code::KeyR);
+                let app_handle = app.handle().clone();
+                let event_app_handle = app_handle.clone(); // Clone for event emission
+
+                match app.handle().plugin(
```

Comment:
can you make this less nested? really hard to read

Reply:
Okay